api/desecapi/models.py

```diff
@@ -389,6 +394,16 @@ class Meta:
     def construct_name(subname, domain_name):
         return '.'.join(filter(None, [subname, domain_name])) + '.'
 
+    @classmethod
+    def construct_catalog_rrset(cls, subname, qtype, content):
```

Comment:
This code is pdns specific and shouldn't live in models.py

Reply:
fix in 2f649f29870a7008e78608be28d0997ba20c0a98